doc/source/whatsnew/v0.18.2.txt

```diff
@@ -154,3 +154,5 @@ Bug Fixes
 - Bug in ``Period`` addition raises ``TypeError`` if ``Period`` is on right hand side (:issue:`13069`)
 - Bug in ``Peirod`` and ``Series`` or ``Index`` comparison raises ``TypeError`` (:issue:`13200`)
 - Bug in ``pd.set_eng_float_format()`` that would prevent NaN's from formatting (:issue:`11981`)
+- Bug in ``Float64Index.astype(int)`` incorrectly converted ``NaN`` to integer (:issue:`13149`)
+- Bug in ``TimedeltaIndex``, ``DatetimeIndex`` where ``.astype(int)`` returned ``np.array`` instead of ``Int64Index`` (:issue:`13209`)
```

Comment:
move these to API changes section.

add that ``Float64Index.astype(int)`` will now raise ``ValueError``